frontend/sentry.js

```diff
@@ -14,7 +14,8 @@ async function createReleaseAndUpload() {
     await cli.releases.uploadSourceMaps(release, {
       include: ['build/static/js'],
       urlPrefix: '~/static/js',
-      validate: true
+      validate: true,
+      rewrite: false,
```

Comment:
It appears that this is true by default starting v1.59.0 [1], and we are currently using v1.54.0. It may be worthwhile to either set this to true, or upgrade to a later version of `@sentry/cli`. I'll look into these options.

[1]: https://github.com/getsentry/sentry-cli/blob/master/CHANGELOG.md#sentry-cli-1590

Reply:
Oh haha , snyk upgraded `@sentry/cli` to the latest version: https://github.com/opengovsg/postmangovsg/pull/1100